jenkins/test-history/gen_json_test.py

```diff
@@ -25,20 +27,50 @@
 import time
 
 
+TEST_BUCKETS_DATA = {
+    'gs://kubernetes-jenkins/logs/': { 'prefix': '' },
+    'gs://bucket1/': { 'prefix': 'bucket1_prefix' },
+    'gs://bucket2/': { 'prefix': 'bucket2_prefix' }
+}
+
+
+class GetBucketsTest(unittest.TestCase):
+    """Tests for gen_json.get_buckets."""
+    def testGetBuckets(self):
```

Comment:
use underscores for test names (@spxtr has a PR that does this for the other tests)